html/about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link href="../css/style.css" rel="stylesheet" type="text/css" />
+    <link href="../css/about.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <!-- hero section -->
+    <div id="home" class="home">
+      <div class="home-top">
+        <div class="top-content">
+          <img src="../assets/facebook.png" alt="facebook" />
+          <img src="../assets/twitter.png" alt="twitter" />
+          <span>Logout</span>
+          <span>MyPage</span>
+          <span>Korean</span>
+        </div>
+      </div>
+      <div class="toolBar">
+        <a href="../index.html"
+          ><img class="logo" src="../assets/medical-logo.png" alt="logo"
+        /></a>
+        <div class="toogle">
+          <img
+            class="mobile-menu"
+            src="../assets/mobile_menu.png"
+            alt="mobile menu"
+          />
+        </div>
+        <nav>
+          <ul class="navigation">
+            <li class="nav-item"><a>About</a></li>
+            <li class="nav-item"><a>Program</a></li>
+            <li class="nav-item"><a>Join</a></li>
+            <li class="nav-item"><a>Sponsor</a></li>
+            <li class="nav-item"><a>News</a></li>
+          </ul>
+          <img
+            class="close-menu"
+            src="../assets/close_mobile_menu.png"
+            alt="close"
+          />
+        </nav>
+        <a class="compain-btn">CC Compain</a>
+      </div>
+      <div class="about">
+        <h1>We Take Care Of Your Healthy Life.</h1>
```

Comment:
- [ ] Here As well According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Layout: composition and space between elements., So kindly Update the color according to the original design and add the secondary heading as show in the given design  with the same font weight also👍💯

- Your design
![image](https://user-images.githubusercontent.com/58553711/197081184-5ee200cf-b919-43c4-a9c3-fcae41603a18.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/197081342-49058a11-51b1-411e-8087-ea037ebf343f.png)

Reply:
Thank you for bringing this to my attention, I will make the necessary updates to match the original design